docker/udp-server/Dockerfile

```diff
@@ -1,13 +1,23 @@
-FROM golang:1
-MAINTAINER Valentin Kuznetsov vkuznet@gmail.com
+FROM golang:1.18 AS build
+LABEL maintainer="Nikodemas Tuckus <tuckus.nikodemas@cern.ch>"
+
+ARG TAG=v00.00.00
+ENV WDIR=/data \
+    CGO_ENABLED=0
+
+WORKDIR $WDIR
+
+RUN git clone https://github.com/dmwm/udp-collector && \
+    cd udp-collector && \
+    if [ "$TAG" = "v00.00.00" ]; then git checkout master; else git checkout tags/$TAG -b build; fi && \
+    make
+
+FROM alpine:3.14
```

Comment:
Since UDP server compiles into static executable I suggest to use distroless image which does not provide any shell in it, and neither contains any Linux stack, it is bare kernel and it tights security to its maximum. We use this for our APS frontends, see https://github.com/dmwm/CMSKubernetes/blob/master/docker/auth-proxy-server/Dockerfile#L19 and comment about with URL explaining what it is.

In new k8s cluster (Aroosha can tell you which version) it supports `kubectl debug` command which will allow to get access to distroless image from another container. Doing this way you image will have always zero vulnerability and since we expose UDP server to outside world CERN security team will welcome that.